src/otx/algo/classification/torchvision_model.py

```diff
@@ -155,45 +154,154 @@ def __init__(
             optimize_gap=True,
         )
 
+    def _get_head(self, net: nn.Module) -> nn.Module:
+        """Returns the head of the model."""
+        head: nn.Module
+        last_layer = list(net.children())[-1]
+        layers = []
+        if self.task == "multiclass":
```

Comment:
Simple question. If `self.task != "multiclass"`, is head not created?

Reply:
Yes, currently only for multi-class tasks. However, i plan to expand this (multi-label, h-label) in the future.